<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfile</class>
 <widget class="QWidget" name="UserProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

font-size:15px;
font-style:MingLiU-ExtB;
}



QFrame#frame{
        background-color: rgba(55, 140, 205, 177);
        background-color: rgba(85, 170, 255, 255);

border-radius:15px;
}
QLineEdit{
color:#8d98a1;
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background-color: rgba(221, 214, 225,0);
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color: rgba(199, 193, 237,0);
border-style:inset;
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
QToolButton#toolButton{
color:rgb(0,0,0);
border-style:none;
border-top-right-radius:15px;
}
QToolButton#toolButton::hover{
color:rgb(255,255,255);
background-color: rgba(255, 56, 56, 0);
}
QToolButton#toolButton_2{
border-style:none;
color:rgb(0,0,0);
}
QToolButton#toolButton_2::hover{
border-style:none;
color:rgb(0,0,0);
background-color: rgba(203, 203, 203, 0);
}
QToolButton#toolButton{
color:rgb(0,0,0);
border-style:none;
border-top-right-radius:15px;
}
QToolButton#toolButton::hover{
color:rgb(255,255,255);
background-color: rgba(255, 56, 56, 150);
}
QToolButton#toolButton_2{
border-style:none;
color:rgb(0,0,0);
}
QToolButton#toolButton_2::hover{
border-style:none;
color:rgb(0,0,0);
background-color: rgba(203, 203, 203, 150);
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>611</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;微软雅黑&quot;;
color:rgb(0,0,0);
</string>
    </property>
    <property name="text">
     <string>更改用户名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeAvatar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>更改头像</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="confirmUsername">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>400</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>完成更新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>更改用户名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>请输入新用户名</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="cancelButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmAvatar">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>400</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>完成更新</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>321</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>请选择新头像</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="cancelButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signatureEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>更改签名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmSign">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>完成更新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
      </property>
      <property name="text">
       <string>请输入新签名</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeSignature">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;幼圆&quot;;
color:rgb(0,0,0);
font-weight:bold</string>
    </property>
    <property name="text">
     <string>更改签名</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
